modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/DefaultImplicitConfigurationProviderFactory.java

```diff
@@ -78,7 +79,7 @@ public ConfigurationProvider createImplicitConfigurationProvider(ExtensionModel
         return new DynamicConfigurationProvider(providerName, extensionModel, implicitConfigurationModel,
                                                 resolverSet,
                                                 implicitConnectionProviderValueResolver,
-                                                ImmutableExpirationPolicy.getDefault(new TimeSupplier()), muleContext);
+                                                ImmutableExpirationPolicy.getDefault(new LocalTimeSupplier()), muleContext);
```

Comment:
What happens when a extension is used in a cluster? Should not use the same time supplier used in the provided muleContext?

Reply:
I double checked with the SDK team, and this is only used for the expiration of the configuration.